prov/efa/src/rxr/rxr_ep.c

```diff
@@ -755,10 +754,11 @@ static int rxr_ep_bind(struct fid *ep_fid, struct fid *bfid, uint64_t flags)
 			return -FI_ENOMEM;
 
 		if (rxr_need_sas_ordering(rxr_ep)) {
-			rxr_ep->robuf_fs = rxr_robuf_fs_create(av->util_av.count,
-							       NULL, NULL);
-			if (!rxr_ep->robuf_fs)
-				return -FI_ENOMEM;
+			ret = ofi_bufpool_create(&rxr_ep->robuf_pool,
+						 sizeof(struct rxr_robuf), 16,
+						 0, 0, OFI_BUFPOOL_NO_TRACK);
```

Comment:
Is `OFI_BUFPOOL_NO_TRACK` needed here? From what I can tell all of the entries will be cleaned up before the bufpool is destroyed.